tasks/os_identification.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
is the whole task required? can't you use https://forge.puppet.com/modules/puppetlabs/facts/readme ?

Reply:
The task works fine without facter/puppet. They don't need to be installed. That's why I suggested it.

Reply:
I still think this should be addressed before merging.